### DISCUSSION TITLE
Front End Shapefile upload

### DISCUSSION THREAD

**Discussion by JosephTheuri** [apache/superset#18280] [Q&A / Help]
I couldn't get to "superset-frontend/src/visualizations/CountryMap/countries" to add a new shape file as indicated in the documentation. Searching for the folder also didn't help

As the documentation is being updated it would be great to allow addition of shape files from the front_end

**Comment by zhaoyongjie** [comment]
Hey @JosephTheuri ,
The map files at this folder: 
https://github.com/apache-superset/superset-ui/tree/master/plugins/legacy-plugin-chart-country-map/src/countries
